src/plugins/intel_cpu/src/plugin.cpp

```diff
@@ -483,12 +484,13 @@ ov::Any Plugin::get_ro_property(const std::string& name, const ov::AnyMap& optio
         return decltype(ov::available_devices)::value_type(availableDevices);
     } else if (name == ov::device::capabilities) {
         std::vector<std::string> capabilities;
-        if (dnnl::impl::cpu::x64::mayiuse(dnnl::impl::cpu::x64::avx512_core_bf16))
+        if (hasHardwareSupport(ov::element::bf16))
```

Comment:
Given https://github.com/openvinotoolkit/openvino/pull/24164/files#r1574670964 I would propose to change on `dnnl::impl::cpu::x64::mayiuse(dnnl::impl::cpu::x64::avx512_core_bf16) || dnnl::impl::cpu::x64::mayiuse(dnnl::impl::cpu::x64::avx2_vnni_2)`

Reply:
fixed